packages/cli/src/executions/execution-recovery.service.ts

```diff
@@ -3,7 +3,6 @@ import { Push } from '@/push';
 import { jsonStringify, sleep } from 'n8n-workflow';
 import { ExecutionRepository } from '@db/repositories/execution.repository';
 import { getWorkflowHooksMain } from '@/WorkflowExecuteAdditionalData'; // @TODO: Dependency cycle
```

Comment:
is this still a Dependency cycle?

Reply:
Unfortunately yes, but pretty small:
- `ExecutionRecoveryService` → Dependency cycle via `@/eventbus/MessageEventBus/MessageEventBus:73`
- `MessageEventBus` → Dependency cycle via `@/WorkflowExecuteAdditionalData:5`